apis/video_api.py

```diff
@@ -59,6 +62,12 @@ def get_car_videos(car , recalls):
                 # append both to the list
                 video_links.append({'title': title, 'embed': embed})
 
+                # if there are no videos returned in the list return a string with  that message
+            if len(video_links) == 0:
+                return 'Car recall videos not found'
+            else:
+                return video_links
```

Comment:
This return statement is inside the loop, so returns after the first recall has been processed

Reply:
Addressed in recent commit